server/src/test/java/org/elasticsearch/search/aggregations/bucket/composite/InternalCompositeTests.java

```diff
@@ -249,6 +251,30 @@ public void testReduceSame() throws IOException {
         }
     }
 
+    /**
+     * Check that reducing with an unmapped index produces useful formats.
```

Comment:
Thank you for including a note of what we're testing, helps so much when debugging test failures!